gbfs.md

```diff
@@ -709,12 +709,12 @@ Field Name | REQUIRED | Type | Defines
-&emsp;&emsp;\-&nbsp;`vehicle_type_id`| Yes | ID | REQUIRED if `vehicle_type_dock_capacity` is defined. A `vehicle_type_id`, as defined in `vehicle_types.json`, that may dock at the station.
-&emsp;&emsp;\-&nbsp;`count`| Yes | Non-negative integer | REQUIRED if `vehicle_type_dock_capacity` is defined. The total number of docks at the station, both available and unavailable, that may accept the corresponding vehicle type as defined by its `vehicle_type_id`.
+\-&nbsp;`vehicle_types_capacity` <br/>*(as of v3.0)* | OPTIONAL | Array | This field's value is an array of objects containing the keys `vehicle_type_ids` and `count` defined below.  These objects are used to model the parking capacity of virtual stations (defined using the `is_virtual_station` field) for each vehicle type that can be returned to this station. The total number of vehicles from each of these objects SHOULD add up to match the value specified in the `capacity` field.
+&emsp;&emsp;\-&nbsp;`vehicle_type_ids` <br/>*(as of v3.0)* | Conditionally REQUIRED | Array | REQUIRED if `vehicle_types_capacity` is defined. The `vehicle_type_ids`, as defined in `vehicle_types.json`, that may park at the virtual station.
+&emsp;&emsp;\-&nbsp;`count`| Conditionally REQUIRED | Non-negative integer | REQUIRED if `vehicle_types_capacity` is defined. A number representing the total number of vehicles of the specified `vehicle_type_ids` that can park within the virtual station.<br /><br />If the virtual station is defined by `station_area`, this is the number that can park within the station area. If `lat`/`lon` is defined, this is the number that can park at those coordinates.
+\-&nbsp;`vehicle_docks_capacity` <br/>*(as of v3.0)* | OPTIONAL | Array | This field's value is an array of objects containing the keys `vehicle_type_ids` and `count` defined below. These objects are used to model the total docking capacity of a station, both available and unavailable, for each type of vehicle that may dock at this station. The total number of docks from each of these objects SHOULD add up to match the value specified in the `capacity` field.
+&emsp;&emsp;\-&nbsp;`vehicle_type_ids` <br/>*(as of v3.0)* | Conditionally REQUIRED | Array | REQUIRED if `vehicle_docks_capacity` is defined. An array of strings where each string represents a `vehicle_type_id` that is able to use a particular type of dock at the station.
+&emsp;&emsp;\-&nbsp;`count`| Conditionally REQUIRED | Non-negative integer | REQUIRED if `vehicle_docks_capacity` is defined. The total number of docks at the station, both available and unavailable, that may accept the corresponding vehicle type as defined by its `vehicle_type_id`.
```

Comment:
The `vehicle_docks_capacity.count` description was referencing the field `vehicle_type_id` instead of `vehicle_type_ids` (plural)

```suggestion
&emsp;&emsp;\-&nbsp;`count`| Conditionally REQUIRED | Non-negative integer | REQUIRED if `vehicle_docks_capacity` is defined. A number representing the total number of docks at the station, both available and unavailable, that may accept the vehicle types specified by `vehicle_type_ids`.
```